tensorflow/tensorflow-0.10.0rc0.patch

```diff
@@ -0,0 +1,25 @@
+diff -ruN tensorflow-master/tensorflow/cc/BUILD tensorflow-master-patch/tensorflow/cc/BUILD
+--- tensorflow-master/tensorflow/cc/BUILD	2016-02-06 10:08:02.000000000 +0900
++++ tensorflow-master-patch/tensorflow/cc/BUILD	2016-02-06 17:42:56.446230724 +0900
+@@ -75,6 +75,21 @@
+     ],
+ )
+
++cc_binary(
++    name = "libtensorflow.so",
```

Comment:
I have no idea, I just copied what was there previously and made minimal changes needed in order to build it. I put up this PR so that we can fix these issues. So thank you for your comments, they are very appreciated, I'll look into it and see what I can improve :)